### PR TITLE
Fix Az.Blueprint Syntax Errors

### PR DESCRIPTION
**Pull request by CaptainFanZzz**
## Description

Fix Az.Blueprint Syntax Errors

## Checklist

- [ ] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [ ] The title of the PR is clear and informative
- [ ] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [ ] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
        - {Please put the link here}
    - [ ] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

File: src/Blueprint/Blueprint/help/Set-AzBlueprintAssignment.md [hunk 1]
```diff
@@ -75,7 +75,7 @@ Update an existing blueprint assignment through an assignment file. The format o
 
 ### Example 3
 ```powershell
-$blueprintObject =  Get-AzBlueprint -ManagementGroup "myManagementGroup" -Name "myBlueprintName"
+$blueprintObject =  Get-AzBlueprint -ManagementGroupId "myManagementGroupId" -Name "myBlueprintName"
 Set-AzBlueprintAssignment -Name "myAssignment" -Blueprint $blueprintObject -ManagementGroupId "myManagementGroup" -SubscriptionId 00000000-1111-0000-1111-000000000000 -Location "West US" -Parameter @{P1="v1"; P2="v2"}
```
Comment (ziyuezh576) [inline src/Blueprint/Blueprint/help/Set-AzBlueprintAssignment.md:79]: Why "myManagementGroupId"?

**Review by ziyuezh576 — CHANGES_REQUESTED** [review]

File: src/Blueprint/Blueprint/help/Set-AzBlueprintAssignment.md (hunk 1 above)
Comment (CaptainFanZzz) [reply src/Blueprint/Blueprint/help/Set-AzBlueprintAssignment.md:79]: Reference Get-Blueprint.md. There is no parameter named ManagementGroup, and its examples are using commonly ManagmentGroupId.

**Review by ziyuezh576 — APPROVED** [review]

File: src/Blueprint/Blueprint/help/Set-AzBlueprintAssignment.md [hunk 2]
```diff
@@ -75,7 +75,7 @@ Update an existing blueprint assignment through an assignment file. The format o
 
 ### Example 3
 ```powershell
-$blueprintObject =  Get-AzBlueprint -ManagementGroup "myManagementGroup" -Name "myBlueprintName"
```
Comment (ziyuezh576) [inline src/Blueprint/Blueprint/help/Set-AzBlueprintAssignment.md:78]: I mean why '-ManagementGroupId "myManagementGroup__Id__"'?

**Review by ziyuezh576 — APPROVED** [review]

**Review by BethanyZhou — APPROVED** [review]

**Pull request merged**